hack/builder/go.mod

```diff
@@ -1,13 +1,110 @@
 module github.com/vmware-tanzu/community-edition/hack/builder
 
-go 1.16
+go 1.17
 
 // -- Used to build TCE plugins with local codebase --
 // replace github.com/vmware-tanzu/tanzu-framework => ../../../tanzu-framework
 
 require (
-	github.com/mattn/go-runewidth v0.0.9 // indirect
 	github.com/spf13/cobra v1.2.1
 	github.com/vmware-tanzu/tanzu-framework v0.10.0
+)
+
+require (
+	cloud.google.com/go v0.81.0 // indirect
```

Comment:
Hmm curious why there are suddenly all the new indirect dependencies throughout? Did something in 1.17 change with how go mod resolves dependencies?

Reply:
Yep, there are changes in the module graph handling. Take a look at the "Pruned module graphs in go 1.17 modules" section in the 1.17 release notes here: https://go.dev/doc/go1.17

Reply:
That's sweet! Good to know!! (and very glad they have made the module graph hopefully less of a pain)